ci/cron/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# Copyright (c) 2019 The DAML Authors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+
+cd "$(dirname "$0")"/../..
+
+eval "$(dev-env/bin/dade assist)"
+
+bazel build //ci/cron:cron
```

Comment:
```suggestion
bazel run //ci/cron:cron
```

Reply:
Thanks.

Reply:
I had to change that back, as `bazel run` seems to run the program from a weird `cwd`.